tests/unit_tests/test_game_rating.py

```diff
@@ -984,10 +1047,426 @@ async def test_single_wrong_report_still_rated_correctly(game: Game, player_fact
+        team1_outcomes = {id: GameOutcome.VICTORY for id in team1}
+        team2_outcomes = {id: GameOutcome.DEFEAT for id in team2}
+        # TODO: When updating from 3.7 to 3.9, use dict union {} | {}
+        assert result.outcomes == {**team1_outcomes, **team2_outcomes}
+
+
+async def test_rating_adjustment_1v1_newbies(rating_service, player_factory):
+    IGNORED = object()
+
+    # Both players have no ratings yet
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={1: {}, 2: {}},
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((1766, 429), abs=1),
+                    2: pytest.approx((1235, 429), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={1: pytest.approx((1766, 429), abs=1)},
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_ladder_newbies_global_pros(
+    rating_service, player_factory
+):
+    IGNORED = object()
+
+    # Both players have no ladder rating, but high global rating
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={
+            1: {RatingType.GLOBAL: Rating(2000, 75)},
+            2: {RatingType.GLOBAL: Rating(1900, 75)},
+        },
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((1766, 429), abs=1),
+                    2: pytest.approx((1235, 429), abs=1),
+                },
+                outcomes=IGNORED
+            )
+        ]
+    )
+
+
+async def test_rating_adjustment_1v1_low_ladder_high_global(
+    rating_service, player_factory
+):
+    IGNORED = object()
+
+    # Both players have lower ladder rating than global, but both are below the
+    # rating adjustment limit
+    await do_test_rating_adjustment(
+        rating_service,
+        player_factory,
+        ratings={
+            1: {
+                RatingType.GLOBAL: Rating(1000, 75),
+                RatingType.LADDER_1V1: Rating(900, 125)
+            },
+            2: {
+                RatingType.GLOBAL: Rating(900, 75),
+                RatingType.LADDER_1V1: Rating(700, 125)
+            },
+        },
+        expected_results=[
+            PersistedResults(
+                rating_type=RatingType.LADDER_1V1,
+                ratings={
+                    1: pytest.approx((923, 122), abs=1),
+                    2: pytest.approx((677, 122), abs=1),
+                },
+                outcomes=IGNORED
+            ),
+            PersistedResults(
+                rating_type=RatingType.GLOBAL,
+                ratings={1: pytest.approx((1006, 75), abs=1)},
```

Comment:
This is actually undesireable behaviour. Because we only adjust for positive changes everybody would over time float to 1400 global even if they are stuck at 500 in the matchmakers.
We need an additional check in the rater to only adjust the rating if it is lower than the matchmaker rating.

Reply:
Ah, yea forgot that one